### PR TITLE
Table style update. Move sort button after title

### PR DESCRIPTION
**Pull request by zombieJ**
### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / document update
- [x] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Branch merge
- [ ] Other (about what?)

### 👻 What's the background?

ref: #15734

### 💡 Solution

From:
屏幕快照 2019-03-31 上午12 11 45

TO:
屏幕快照 2019-03-31 上午12 11 34

### 📝 Changelog description

1. Move Table sort button after title closely.

2. 将 Table 排序按钮调至标题后。

### ☑️ Self Check before Merge

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/table/style/index.less [hunk 1]
```diff
@@ -69,30 +69,43 @@
...
-      &-up,
-      &-down {
-        .iconfont-size-under-12px(11px);
-        display: block;
-        height: 4px;
-        line-height: 4px;
+      .@{table-prefix-cls}-column-sorter-inner {
+        height: 1em;
+        margin-top: 0.4em;
+        margin-left: 0.3em;
```
Comment (afc163) [inline components/table/style/index.less]: 可以远一点点。

**Comment by afc163** [comment]
改完以后好像上下两个箭头的距离远了一点？感觉垂直上也有点偏下。

**Comment by zombieJ** [comment]
👌上下微调了 0.05em，左侧 4px 改成 8px：
屏幕快照 2019-03-31 上午12 31 11

**Comment by afc163** [comment]
上下箭头的间距也调整一下。

**Comment by zombieJ** [comment]
上下间距比以前小了一点点，我调整一下：
屏幕快照 2019-03-31 上午9 43 22

**Comment by zombieJ** [comment]
done. Pls check:
屏幕快照 2019-03-31 上午9 49 22

**Review by afc163 — APPROVED** [review]

**Comment by creeve** [comment]
When the column sorter is not shown, the empty sorter span is still rendered (<span class="ant-table-column-sorter"></span>), but since it's parent div does not have the "ant-table-column-sorters" class, that div does not get the "display: table" style applied, and the sorter span is rendered on a new line.  This makes the header much taller than it should be.

Either the parent div should have display: table, or the "ant-table-column-sorter" span should not be rendered.

**Comment by zombieJ** [comment]
hi @creeve , could you help to create an issue for this?

**Comment by creeve** [comment]
@zombieJ - Oops, I was wrong.  Not sure how I missed it, but my issue was caused by my own custom styling left over from when the sorter was displayed on the far right side of the header.  Next time I will create a reproduction before commenting!  Sorry!

**Comment by zombieJ** [comment]
That's OK. You are welcome : )

**Pull request merged**